docs/development/changelogs/writing-changelogs.md

```diff
@@ -0,0 +1,241 @@
+# Writing changelogs
+
+In all of our repositories (except for the solely documentation ones) we have a
+`CHANGELOG.md` file that contains notes written by the developers who
+implemented the respective change.
+
+This document is a guide on how to update those files when you yourself create
+a pull request.
+
+To get an overview of how they look, as well as our reasoning to why we're
+using changelogs the way we do, we suggest that you check out our guide on 
+[reading changelogs](development/changelogs/).
+
+## Prerequisites
+
+- Get proficient with the Markdown syntax. The "GitHub Flavored Markdown" (GFM)
+  variant, to be exact, as there exists many flavors of Markdown.
+
+  Recommended reading: [Mastering Markdown - GitHub Guides](https://guides.github.com/features/mastering-markdown/)
+
+- Use a Markdown linter in your text editor of choice. We have strict rules for
+  what is recognized as valid Markdown, but improperly formatted. We use
+  [remark-lint](https://github.com/remarkjs/remark-lint) in our continuous
+  integration pipeline to validate our Markdown files, including our
+  `CHANGELOG.md` files.
+
+  Recommended to check out their integrations guides: [Integrations - remark-lint](https://github.com/remarkjs/remark-lint#integrations)
+
+- We're using [SemVer 2.0.0](https://semver.org/) in all of our versioned
+  repositories. You are expected to follow the SemVer 2.0.0 rules when
+  potentionally bumping the version in the `CHANGELOG.md` file, and to
```

Comment:
unexpected portmanteau detected
I think it can be skipped entirely though, feels a bit superfluous.
```suggestion
  potentially bumping the version in the `CHANGELOG.md` file, and to
```

Reply:
What words is portmanteau'd?